libr/core/core.c

```diff
@@ -258,15 +258,17 @@ static ut64 num_callback(RNum *userptr, const char *str, int *ok) {
 		}
 		// pop state
 		if (ok) *ok = 1;
-		ut64 num = 0;
+		ut8 buf[sizeof(ut64)];
+		(void)r_io_read_at (core->io, n, buf, refsz);
```

Comment:
maybe will be good to check for return value here, or at least initialize buf to avoid parsing uniniitalized data